service/src/main/java/cz/muni/fi/pa165/soccermanager/service/ManagerService.java

```diff
@@ -1,17 +1,25 @@
 package cz.muni.fi.pa165.soccermanager.service;
 
 import cz.muni.fi.pa165.soccermanager.entity.Manager;
+import cz.muni.fi.pa165.soccermanager.entity.Team;
+import cz.muni.fi.pa165.soccermanager.enums.NationalityEnum;
 
 import java.util.List;
 
 /**
- * @author 445720 Martin Hamernik
- * @version 11/16/2017.
+ * An interface that defines a service access to the Manger entity
+ * @author 456519 Filip Lux
+ * @version 11/23/2017.
  */
 public interface ManagerService {
     Manager fetchById(long managerId);
+    Manager fetchByName(String name);
+    Manager fetchByEmail(String email);
+    Manager fetchByTeam(Team team);
+    List<Manager> fetchByNationality(NationalityEnum nationalityEnum);
+    List<Manager> fetchManagersWithoutTeam();
     List<Manager> fetchAll();
-    void insert(Manager manager);
+    void create(Manager manager);
     void update(Manager manager);
-    void delete(long managerId);
+    void remove(long managerId);
```

Comment:
I would stick with insert and delete so we are consistent in whole project. But we can discuss this and refactor it other classes too